cybersecuritytools/aws_permissions_helper/README.md

```diff
@@ -0,0 +1,40 @@
+# Aws Permissions Helper 
+
+This script will read the a logfile and extract all the requests made to create the resources and output them to a file called `aws_requests.txt`
+
+## Usage
+
+Firstly, you will need to enable the correct logging level by terraform, before the next step run:
+
+```bash
+export TF_LOG=TRACE
```

Comment:
Presumably if your pattern is looking for DEBUG log lines you can set `export TF_LOG=DEBUG` and make the log file smaller?